app/controllers/proxy_controller.rb

```diff
@@ -1,17 +1,15 @@
 class ProxyController < ApplicationController
-  def configuration
-    @config ||= ::Configuration.new(okapi).tap do |config|
-      config.load!
-    end
-  end
+  before_action :slice_eholdings_namespace
```

Comment:
Let's compute the desired path with a simple function instead of an action that, as one of its side effects, mutates `request.fullpath`

Reply:
my original intent was to set this to an instance variable, but i guess i never got around to correcting that.  your proposed approach is better either way, so i'll go that route.